content/articles/monitoring-microsoft-sql-server-using-solar-windows-database-analyzer/index.md

```diff
@@ -0,0 +1,158 @@
+### Introduction
+Databases hold a lot of critical and highly sensitive data for various organizations. For this reason, it is crucial to monitor them and hence make sure they are securely accessed. The article will aim at enlightening the reader on how to monitor a Microsoft SQL Server database using Solar Winds Database Performance Analyzer (Self-Managed). This is a software offered by Solar Winds and is very useful in attaining the security of a database.
+
```

Comment:
Can you talk more about the goals of this tutorial? What are the key takeaways? Objective of the code tutorial?